tasks/Logger.cpp

```diff
@@ -349,3 +367,50 @@ void Logger::snapshot()
         this->engine()->getActivity()->trigger();
 }
 
+void Logger::renameFile()
+{
+    // create timestamp
+    time_t now = time(0);
+    tm *t_ptr = localtime(&now);
+    char suffix[21];
+    strftime(suffix, sizeof(suffix), "%F_%H-%M-%S", t_ptr);
+    // append suffix to previous _current_file.get()
+    vector<string> strs;
+    boost::split(strs, _file.value(), boost::is_any_of("."));
+    if ( strs.size() == 1) {
+        strs.insert(strs.end(), std::string(suffix));
+    } else {
+        strs.insert(strs.end()-1, std::string(suffix));
+    }
```

Comment:
Would be good to add a test for this one ...

Reply:
Done :+1: